src/services/utils.js

```diff
@@ -0,0 +1,147 @@
+// @flow
+'use strict'
+
+import $localForage from 'localforage'
+import _cloneDeep from 'lodash.clonedeep'
+import _startsWith from 'lodash.startswith'
```

Comment:
Can we change this to use the native String.prototype.startsWith https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith please

Reply:
yeop.